include/openssl/ssl.h

```diff
@@ -894,7 +899,9 @@ typedef enum {
     TLS_ST_SW_KEY_UPDATE,
     TLS_ST_CW_KEY_UPDATE,
     TLS_ST_SR_KEY_UPDATE,
-    TLS_ST_CR_KEY_UPDATE
+    TLS_ST_CR_KEY_UPDATE,
+    TLS_ST_EARLY_DATA,
+    TLS_ST_PENDING_EARLY_DATA_END
```

Comment:
It's only actually a problem if the values go >MAX_INT right? Which seems quite unlikely. Changing it in 1.1.0 now could break stuff - so it doesn't seem justified to me. Probably a discussion for the team rather than here though.